### PR TITLE
Submit front page form to UsersController

### PR DESCRIPTION
**Pull request by jfranciscosousa**
Why:

* The form in the front page should be the main source of registrations

This change addresses the need by:

* Changing input names to match the controller required params
* Disabling CSRF in the UsersController
* Enabling CORS to make up for the CSRF loss (only on production)
* Creating the `CreateUserPage`, that is displayed after registrations

### CODE REVIEW

File: app/views/users/error.slim [hunk 1]
```diff
@@ -0,0 +1 @@
+= render_react_component(component: "UserCreationError", prerender: true)
```
Comment (luistorres) [inline app/views/users/error.slim]: empty line

File: frontend/components/Layout/index.css [hunk 2]
```diff
@@ -0,0 +1,14 @@
+@import '../../styles/breakpoints.css';
+@import '../../styles/layout.css';
+
+.root {
+  @include Breakpoint-desktopOnly {
+    max-width: $content-width;
+
+    margin: $spacing-112 auto $spacing-112 auto;
```
Comment (luistorres) [inline frontend/components/Layout/index.css]: `margin: $spacing-112 auto`

File: app/views/users/error.slim (hunk 1 above)
Comment (jfranciscosousa) [reply app/views/users/error.slim]: Fixed

File: frontend/components/Layout/index.css (hunk 2 above)
Comment (jfranciscosousa) [reply frontend/components/Layout/index.css]: Fixed

**Review by frm — APPROVED** [review]

**Pull request merged**